content_sync/tasks.py

```diff
@@ -149,6 +149,11 @@ def preview_website_backend(website_name: str, preview_date: str):
         backend = api.get_sync_backend(website)
         backend.sync_all_content_to_backend()
         backend.create_backend_preview()
+
+        pipeline = api.get_sync_pipeline(website)
+        build_id = pipeline.trigger_pipeline_build(BaseSyncPipeline.VERSION_DRAFT)
+        Website.objects.filter(pk=website.pk).update(latest_build_id_draft=build_id)
```

Comment:
Works fine as is, but might seem more logical to place these lines after the two below it (unpause pipeline).